frontend/src/app/components/LocalizedHtml.js

```diff
@@ -0,0 +1,80 @@
+
```

Comment:
Might need to soak my head in this component for a bit longer to understand the why & how